.goreleaser.yml

```diff
@@ -14,6 +14,7 @@ builds:
       - linux_amd64
       - linux_arm64
       - windows_amd64
+      - windows_arm64
```

Comment:
I'd rather not support another platform when there's no one in the team with a Windows ARM machine. I don't see anyone requesting for it either.

Reply:
This is the ticket which requests for windows arm support https://github.com/supabase/cli/issues/1315. Will move to a separate PR though.